maxun-core/src/interpret.ts

```
@@ -548,232 +548,276 @@ export default class Interpreter extends EventEmitter {
-                }
-                previousHeight = currentHeight;
-                
-                if (config.limit && allResults.length >= config.limit) {
-                  allResults = allResults.slice(0, config.limit);
-                  break;
-                }
-              }
-              break;
+              if (!loadMoreSuccess || checkLimit()) return allResults;
+            }
+          }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)